doc/source/turkey.md

```diff
@@ -0,0 +1,33 @@
+# 🇹🇷 Turkey
+
+Turkey, officially known as the Republic of Turkey, is the country which is the bridge between Europe and Asia. It shares borders with 8 different countries and 3 seas. It consists of 7 geographical regions and 81 cities. The region on which the country is situated is called Anatolia and accepted as a peninsula. Anatolia has been a strategically important area throughout the history. Turkish people came to Anatolia in 1071 first and since then they are living there. Never mind, let’s focus on the country again.
+
+The country has a population over 80 million and Kurds are the largest minority with a population over 16 million. Since the beginning of Syrian Civil War, Turkey has been a shelter to Syrians and there are over 2 million Syrian refugees in Turkey. Even though there are ethnicities other than Turkish in the country, every Turkey citizen is accepted Turkish by law. That does not mean Turkey forces people to forget about their roots. The law has been put into order to ensure that every citizen is treated equally.
+
+The country’s official language is Turkish. Kurdish is the second most spoken language in the country, but it is not accepted as an official language, there are people who supports this idea, though.
+
+Even though, due to being the most populous city in Turkey, Istanbul is thought to be the capital of the country, the country’s capital is Ankara since the law about the country’s capital was accepted in 1923 by the new parliament of the new country.
+
+If we are to talk about a country, we should talk about their economy. Turkey has a GDP of $794 billion in total which is the 20th biggest one in the world but when you consider the population (GDP per capita) Turkey is 67th in the world. Also, the purchasing power of the Turkish population is the 45th best one in the world.
+
+One of the biggest industries in Turkey is tourism. It has been so beneficial to economy of the country that the tourism has been called “smokeless industry”. Turkey ranked sixth in the world in terms of the number of international tourist arrivals, with over 50 million foreign tourists visiting the country.
```

Comment:
One of the biggest industries in Turkey is tourism. It has been so beneficial to the economy of the country that tourism has been called a “smokeless industry”. Turkey ranked sixth in the world in terms of the number of international tourist arrivals, with over 50 million foreign tourists visiting the country.